appveyor.yml

```diff
@@ -46,7 +49,6 @@ test_script:
   - bundler --version
   - bash ./script/cibuild
 
-cache:
+#cache:
   # If one of the files after the right arrow changes, cache will be skipped
-  - C:\Ruby%RUBY_FOLDER_VER%\bin -> Gemfile,jekyll.gemspec,appveyor.yml
-  - C:\Ruby%RUBY_FOLDER_VER%\lib\ruby\gems\%GEMS_FOLDER_VER% -> Gemfile,jekyll.gemspec,appveyor.yml
+#  - 'vendor -> appveyor.yml,Gemfile,jekyll.gemspec'
```

Comment:
We can't use a custom deployment location because bundler complains that there is no Gemfile.lock. See https://ci.appveyor.com/project/jekyll/jekyll/build/259

I'm no expert with Rubygems so perhaps there is another way to do it instead of using `bundler --deployment`.